Source/cursor.cpp

```diff
@@ -131,22 +131,17 @@ void SetICursor(int i)
 	icursH28 = icursH / 28;
 }
 
-void SetCursor_(int i)
+void NewCursor(int i)

```

Comment:
Not sure how this is related to enums. Would be nice to avoid touching unrelated things.

Reply:
It was right there in front of me when doing the enums.